packages/twenty-front/src/modules/views/components/ViewBarDetails.tsx

```diff
@@ -29,21 +29,23 @@ const StyledBar = styled.div`
   border-top: 1px solid ${({ theme }) => theme.border.color.light};
   display: flex;
   flex-direction: row;
-  min-height: 32px;
+  align-items: center;
   justify-content: space-between;
+  position: relative;
+  padding: ${({ theme }) => theme.spacing(1)};
   z-index: 4;
-  padding-top: ${({ theme }) => theme.spacing(1)};
-  padding-bottom: ${({ theme }) => theme.spacing(1)};
+  min-height: 32px;
 `;
 
 const StyledChipcontainer = styled.div`
   align-items: center;
   display: flex;
   flex-direction: row;
   gap: ${({ theme }) => theme.spacing(1)};
-  min-height: 32px;
-  margin-left: ${({ theme }) => theme.spacing(2)};
-  flex-wrap: wrap;
+  overflow-x: scroll;
```

Comment:
I'm not sure about this scroll behavior, it's very difficult to use. I think we want they to pile vertically
@Bonapara could confirm

Reply:
let's remove the scroll

Reply:
@charlesBochet, In the figma designs and the video in #7018 the filter chips scroll vertically behind the Reset and Save as new view state. If we remove this scroll then how should we approach it ?

Reply:
oh ok! I missed that, I feel this is not easy to use and need to be discussed again with Bonapara for small screens but we could merge the PR

Reply:
@charlesBochet, I agree to your point. To overcome this, can we can do a tooltip or information badge for all mobile viewports as a result when user first time applies filters, they can learn ?